apps/nextjs/src/components/DialogControl/DialogContents.tsx

```diff
@@ -67,7 +67,13 @@ const DialogContents = ({
       <OakModalCenterBody
         title={dialogTitlesAndIcons[dialogWindow].title}
         iconName={dialogTitlesAndIcons[dialogWindow].iconName}
-        hideIcon={dialogWindow === "feedback" || dialogWindow === "share-chat"}
+        iconOverride={
+          dialogWindow === "feedback" || dialogWindow === "share-chat"
+            ? {
+                $display: "none",
+              }
+            : {}
+        }
```

Comment:
`hideIcon` doesn't seem to exist. This seems to do the job

Reply:
You might need to run `npm install` (to get the later version of oak components)

Reply:
oh! I'll have a look

Reply:
Correct! Brownie points for you. Thanks